### ISSUE TITLE
tests for perl library `DBIx-Class` fail (which causes `hydra-unstable` to not be able to be built)

### ISSUE THREAD

**Issue by cdepillabout** [NixOS/nixpkgs#152319]
### Describe the bug

It appears that the tests for the Perl library `DBIx-Class` are failing on `master`.  I noticed this when trying to use `hydra-unstable`:

```console
$ nix repl ./.
nix-repl> hydra-unstable
«derivation /nix/store/hzp3bpdxvh9srz8yxx46sp3kwyqayrll-hydra-2021-08-11.drv»
nix-repl> :b hydra-unstable
error: builder for '/nix/store/2hk7n3zl8fa57d1li0hjgdmmcci57g8m-perl5.34.0-DBIx-Class-0.082842.drv' failed with exit code 2;
       last 10 log lines:
       >   Non-zero exit status: 2
       > t/64db.t                                          (Wstat: 256 Tests: 4 Failed: 1)
       >   Failed test:  4
       >   Non-zero exit status: 1
       > t/sqlmaker/limit_dialects/generic_subq.t          (Wstat: 0 Tests: 19 Failed: 0)
       >   TODO passed:   9, 11, 13, 15, 17
       > Files=325, Tests=42256, 103 wallclock secs ( 2.07 usr  0.78 sys + 73.24 cusr 27.71 csys = 103.80 CPU)
       > Result: FAIL
       > Failed 2/325 test programs. 3/42256 subtests failed.
       > make: *** [Makefile:1370: test_dynamic] Error 255
       For full logs, run 'nix log /nix/store/2hk7n3zl8fa57d1li0hjgdmmcci57g8m-perl5.34.0-DBIx-Class-0.082842.drv'.
error: 1 dependencies of derivation '/nix/store/vq6g113lj0mz1g8cqy8xckli3a0z545c-hydra-perl-deps.drv' failed to build
error: 1 dependencies of derivation '/nix/store/hzp3bpdxvh9srz8yxx46sp3kwyqayrll-hydra-2021-08-11.drv' failed to build
```

This is on commit c7be038ad1da4196f0 (which is `master` as of 2021-12-27).

I can give the full build output if necessary.

### Notify maintainers

Unfortunately it doesn't look like `DBIxClass` itself has any maintainers.  I'm not sure who the main Perl maintainers are.

`hydra-unstable` also doesn't have any maintainers, but the NixOS Hydra tests do have some maintainers: @pstn @nlewo @Ma27 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.81, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.5pre20211126_55275fc`
 - channels(root): `"nixos-21.11.334247.573095944e7"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by pstn** [comment]
Huh. First Time I've heard about being a maintainer of anything hydra. Seems like I was added in the first iteration of that test. I have no idea why and I'm going to go ahead and remove myself.

@nlewo Do you know what happened here? https://github.com/NixOS/nixpkgs/commit/47fc27b4562042bc309ff369c0369f001adb9ef0

**Comment by veprbl** [comment]
Should be fixed by #152207

**Comment by cdepillabout** [comment]
Thanks, looks like this is fixed by #152207.
